src/index.js

```diff
@@ -13,15 +14,52 @@ const traverse = require('traverse');
 
 const replacement = '--REDACTED--';
 
+/**
+ * Apply serializers.
+ */
+
+function applySerializers(values, serializers) {
+  for (const key in serializers) {
+    if (values[key] === undefined) {
+      return;
+    }
+
+    try {
+      values[key] = serializers[key](values[key]);
+    } catch (error) {
+      values[key] = `Anonymize ERROR: Error while applying ${key} serializer: ${error.message}`;
```

Comment:
Wouldn't it be better to return the original value, and log a warning? These serializers should never throw, though.

Reply:
If the property is set to be anonymized it is best to follow this approach, or similar, instead of returning the original value. Otherwise we may be compromising secure data like tokens, secrets, etc.